content/en/tracing/legacy_app_analytics/_index.md

```diff
@@ -173,7 +173,7 @@ App Analytics can be enabled for specific integrations.
 To do so, set either `DD_<INTEGRATION>_ANALYTICS_ENABLED=true` in your environment, or configure with:
```

Comment:
⬆️  In line 52, I believe you missed changing `Datadog.configure { |c| c.analytics_enabled = true }` to `Datadog.configure { |c| c.tracing.analytics.enabled = true }`

Reply:
Nice catch!